lmfit/minimizer.py

```diff
@@ -1479,10 +1480,11 @@ def leastsq(self, params=None, **kws):
         try:
             lsout = scipy_leastsq(self.__residual, variables, **lskws)
             _best, _cov, infodict, errmsg, ier = lsout
-            result.residual = infodict['fvec']
+            # result.residual = infodict['fvec']
         except AbortFitException:
             pass
 
+        result.residual = self.__residual(_best)
```

Comment:
see above, adding ```result.nfev -= 1``` would be good

Reply:
the variable ```_best``` doesn't exist if the fit is aborted... so the current logic doesn't work anymore